src/backend/common/datafeeds/parsers/fms_api/fms_api_event_alliances_parser.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any, Dict, List
+from backend.common.datafeeds.parsers.parser_base import ParserBase
+
+
+class FMSAPIEventAlliancesParser(ParserBase[List[Dict[str, Any]]]):
```

Comment:
We should look to see if we have a type alias for this sort of thing - or maybe expose one here? I'm sure we do this all over for alliances.

Reply:
Ah yeah sure enough - see [`EventAlliance`](https://github.com/the-blue-alliance/the-blue-alliance/blob/py3/src/backend/common/models/alliance.py)